src/arrayMethodSort.js

```diff
@@ -3,9 +3,40 @@
 /**
  * Implement method Sort
  */
+
+const compareStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
```

Comment:
Try to avoid `else if` and `else` statements.
```
if (stringA > stringB) {
    return 1;
} 
if (stringA === stringB) {
   return 0;
}
return -1;
```